src/neural/encoder.cc

```diff
@@ -35,17 +35,136 @@ namespace {
 const int kMoveHistory = 8;
 const int kPlanesPerBoard = 13;
 const int kAuxPlaneBase = kPlanesPerBoard * kMoveHistory;
+
+int CompareTransposing(uint64_t value, int initial_transform) {
```

Comment:
I guess it's more readable to have `ChessBoard` (or is it `BitBoard`) param rather than `uint64_t`.
Also will reduce amount of `as_int()` in the calling code.

Reply:
Yeap, I recall thinking much the same myself, but forgot to come back to it.
Done.

Reply:
Not really done. :)

Reply:
Looks done to me - did you miss a 'refresh' ?